api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java

```diff
@@ -186,4 +186,36 @@ public void testDHISMappingExport() throws IOException {
 				dhisConnectorService.exportMappings(new String[]{"mapping-test-unit." + mapping.getCreated()});
 		Assert.assertEquals("Successfully bundled the mapping with the metadata", pathToBundle[0]);
 	}
+
+	@Test
+	public void testDHISMappingImport() throws IOException {
+		DHISConnectorService dhisConnectorService = Context.getService(DHISConnectorService.class);
+		ReportDefinitionService reportDefinitionService = Context.getService(ReportDefinitionService.class);
+
+		DHISMapping mapping = new DHISMapping();
+		mapping.setName("mapping-test-unit");
+		mapping.setCreated(new Date().getTime());
+		PeriodIndicatorReportDefinition pir = new PeriodIndicatorReportDefinition();
+		final UUID pirUuid = UUID.randomUUID();
```

Comment:
Did you declare `pirUuid` just to pass it in `String.valueOf`?

Reply:
Yup, I can just pass `UUID.randomUUID()` straight into the next string varaiable

Reply:
Updated